util/crc32c_arm64.cc

```diff
@@ -12,6 +12,26 @@
 #ifndef HWCAP_CRC32
 #define HWCAP_CRC32 (1 << 7)
 #endif
+
+#ifdef HAVE_ARM64_CRYPTO
+/* unfolding to compute 8 * 3 = 24 bytes parallelly */
+#define CRC32C24BYTES(ITR) \
+  crc1 = crc32c_u64(crc1, *(buf64 + BLK_LENGTH + (ITR)));\
+  crc2 = crc32c_u64(crc2, *(buf64 + BLK_LENGTH*2 + (ITR)));\
+  crc1 = crc32c_u64(crc1, *(buf64 + (ITR)));
```

Comment:
Both of line 19 and line 21 uses crc1. Won't it have data dependency, which limits 3-way computation? Any reason we moved away from crc0?

Reply:
Thanks for your review. That maybe a coding mistake in line21 and I have fixed it.

Reply:
I am very confused about why we can get better performance results when it cannot make use of 2-way CRC. I'm also curious why unit tests still pass with the bug. Is it because that our unit test doesn't cover enough cases to cover the bug, or is it because they coincidentally generate the same CRC results despite the bug?

Reply:
I'm confused about that. I guess Maybe the CI system is x86 platform and the arm64 branch PR is filtered in compiling.

Reply:
If the CI system supports cross compiling, unit test binary files also need to run on arm64 platform to verify the PR.

Reply:
When you run on your ARM host, did you see crc32c_test passing with the bug you just fixed?

Reply:
If line 21 uses crc1, crc32c_test will be failed. Conversely, crc32c_test will pass when using crc0.